source/controller/wizard_controller.py

```diff
@@ -370,19 +370,11 @@ def on_button_method_clicked(self):
         self.method_page.completeChanged.emit()
         self.wizard_window.next_button.setToolTip('')
 
-    """
     def on_update_graph_file(self):
-        file_dialog = QFileDialog()
-        file_dialog.setNameFilters(["Text files (*.txt *.txt.gz)", "Graph6 File (*.g6 *.g6.gz)"])
-        self.graph_files_page.tr("Text files (*.g6)")
-        file_path = file_dialog.getOpenFileName(
-            filter="Graph6 Files (*.g6 *.txt *.g6.gz *.txt.gz);;"
-                   "Text files (*.txt *.txt.gz);;"
-                   "Graph6 files (*.g6 *.g6.gz)"
-        )
-
-        self.save_graph_file_path(file_path[0], input_file[-1])
-    """""
+        temp1 = self.graph_files_page.list_files_input.count()
```

Comment:
What is `temp1`? Is this a descriptive variable name?

Reply:
Sorry, the comment and `temp1` are garbage that I ended up bringing back from another solution I tried before